readme.md

```diff
@@ -66,14 +87,12 @@ $ strip-css-comments --help
     $ strip-css-comments < src/app.css --all
 ```
 
-
 ## Benchmark
 
 ```
 $ npm run bench
 ```
 
-
```

Comment:
Why did you remove these lines? These are consistent and there because the author chose to do so. Don't change unrelated things in this PR ;)